sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -1604,6 +1611,7 @@ public void onClick(DialogInterface dialog, int which) {
 
         /* Don't show this dialog again. */
         SharedPreferencesManager.remove(PREFERENCE_KEY_UPDATE_SETUP_FAILED_MESSAGE_KEY);
+
```

Comment:
Revert pls. Normally we only add empty lines before a comment.

Reply:
Done